esphome/components/mcp4461/output/mcp4461_output.cpp

```diff
@@ -39,15 +46,15 @@ void Mcp4461Wiper::turn_off() { this->parent_->disable_wiper_(this->wiper_); }
 void Mcp4461Wiper::increase_wiper() {
   if (this->parent_->increase_wiper_(this->wiper_)) {
     this->state_ = this->update_state();
-    ESP_LOGV(TAG, "Increased wiper %d to %" PRIu16 "", this->wiper_,
+    ESP_LOGV(TAG, "Increased wiper %hhu to %" PRIu16 "", static_cast<uint8_t>(this->wiper_),
```

Comment:
```suggestion
    ESP_LOGV(TAG, "Increased wiper %" PRIu8 " to %" PRIu16 "", static_cast<uint8_t>(this->wiper_),
```

I think that `hhu` is only needed for scanf. To be safe, best use PRIu8